packages/hash/search-loader/src/index.ts

```diff
@@ -22,6 +22,13 @@ import { StatsD } from "hot-shots";
 import { getSystemAccountId, getEntityType } from "./db";
 import { logger, INSTANCE_ID } from "./config";
 
+const OPENSEARCH_ENABLED = process.env.HASH_OPENSEARCH_ENABLED === "true";
+if (!OPENSEARCH_ENABLED) {
+  // eslint-disable-next-line no-console
+  console.log("Opensearch is not enabled. Shutting down search-loader");
+  process.exit(1);
```

Comment:
```suggestion
  process.exit(0);
```

`1` means error, but we do as expected here.